src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -420,15 +420,19 @@ private function registerWorkflowConfiguration(array $workflows, ContainerBuilde
             }
 
             // Create a Definition
-            $definitionDefinition = new Definition(Workflow\Definition::class);
-            $definitionDefinition->setPublic(false);
-            $definitionDefinition->addArgument($workflow['places']);
-            $definitionDefinition->addArgument($transitions);
-            $definitionDefinition->addTag('workflow.definition', array(
-                'name' => $name,
-                'type' => $type,
-                'marking_store' => isset($workflow['marking_store']['type']) ? $workflow['marking_store']['type'] : null,
-            ));
+            $definitionDefinition = (new Definition(Workflow\Definition::class))
+                ->setPublic(false)
+                ->addArgument($workflow['places'])
+                ->addArgument($transitions)
+                ->addTag('workflow.definition', array(
+                    'name' => $name,
+                    'type' => $type,
+                    'marking_store' => isset($workflow['marking_store']['type']) ? $workflow['marking_store']['type'] : null,
+                ))
+            ;
```

Comment:
Not sure if it's really worth making this change. It makes `git blame` pretty useless when you try to figure out when a certain change was introduced without much value.